37.md

```diff
@@ -0,0 +1,35 @@
+NIP-37
+======
+
+Transport methods announcement
+-----------------------------------
+
+`draft` `optional`
+
+This NIP describes how digital services can announce how clients can reach them, it describes which network is used and depending on the network what other data is needed to reach it.
```

Comment:
What do you mean by "digital service"?

Reply:
Any service/server really. Some examples would be Cashu Mints, Relays,  Epoxy Proxies, IOT devices...